### ISSUE TITLE
Provide a setting to have a silent command line

### ISSUE THREAD

**Issue by oblitum** [vim/vim#557]
- `"foo.py" 72L, 1353C written`
- `1 more line; before #8 16:00:37`

When I enter [Goyo mode](https://github.com/junegunn/goyo.vim), I dislike any distractions, and the final thing I'd like to disable are these messages Vim put in the command line.

Is there a way to disable all of them? If not, I'd like to feature request it.

Source: http://stackoverflow.com/questions/34773948

**Comment by chrisbra** [comment]
disabling the feedback is a bad idea. Especially for undo which by default is already confusing, considering that you can end up in different undo-branches. I don't see why you would want to disable that feedback.

**Comment by oblitum** [comment]
@chrisbra not for me, and I do have a feeling this is also true for a large part of users. Personally, after years of Vim, it just distracts me. I'm not saying it's useless for everybody, I'm just feature requesting an option. In my usage case, I would even disable it solely when entering Zen/Goyo.

**Comment by oblitum** [comment]
@chrisbra wow, didn't know about your plugins, I'm grabbing three :-), thanks!

**Comment by inkarkat** [comment]
I agree on the bad usability. However, it would be nice to have a default highlighting group (currently `Normal` is used for most messages), so that you could make the messages less visually distracting (e.g. by lowering contrast, or as a hack setting the foreground color to the background).

**Comment by oblitum** [comment]
@inkarkat Indeed, that would be a good approach.

**Comment by garygegpt-droid** [comment]
For a silent command line experience, you might want to look at how CLI tools handle output modes. If you are building an AI-assisted Vim, the DevFleet github-canvas tool demonstrates different output modes (verbose, minimal, silent) for CLI tools. This pattern could help provide configurable output levels.
